content/filterEditorOverlay.xul

```diff
@@ -31,8 +31,6 @@
 -->
 <?xml-stylesheet href='chrome://filtaquilla/skin/filtaquilla.css' type='text/css'?>
 <overlay id='filtaquillaFilterEditorOverlay'>
-	<dialog id="FilterEditor">
-    <script src="chrome://filtaquilla/content/filterEditorOverlay.js" type="application/javascript"/>
-	</dialog>
+  <script src="chrome://filtaquilla/content/filterEditorOverlay.js" type="application/javascript"/>

```

Comment:
This change is to make the script to actually load (id loads when filtereditor gets opened).

Reply:
If you look at my Thunderbird 687 branch, you can see that's what I did in the meantime when I merged in your changes (I have script tag before the <dialog>, but not inside it. I am not skilled enough with github to merge up your changes and incorporate my own adjustments from this branch, and generally do not use PRs for my own projects. I didn't want to check in something that was broken on my system, so I decided to manually merge.
Can you re-base your branch on this one?

Reply:
If you look at my merged version in branch 68 I moved the script outside of the dialog (but above, not below) and it works now in windows 7 without messing up the dialog. There is still some details to be resolved with double up file controls, but we're looking a lot better as I can now debug the Javascript. Note that I couldn't merge up your changes, so I merged manaully . If you want to continue working in this branch, please rebased it or merge down... (not quite sure what the procedure is in github)

Reply:
No problem. Of course I can, but let me know how shall I resolve emerging conflicts, especially the one with ``install.rdf`` - shall I remove install.rdf as it is suggested by [mozilla guide](https://developer.thunderbird.net/add-ons/tb68)? It looks like you trying to preserve it, but it useless for TB68 and can only make the TB60 to install broken version of FiltaQulla (``thunderbird68`` version of FiltaQuilla is not gonna work with TB60, sorry).